service/prometheus/prometheus.go

```diff
@@ -5,11 +5,17 @@ import (
 )
 
 const (
+	// ClusterAnnotation is the Kubernetes annotation that identifies Services
+	// that the prometheus-config-controller should scrape.
 	ClusterAnnotation = "giantswarm.io/prometheus-cluster"
 
 	// ClusterLabel is the Prometheus label used to identify jobs
 	// managed by the prometheus-config-controller.
 	ClusterLabel = "prometheus_config_controller"
+
+	// ClusterIDLabel is the Prometheus label used to identify guest cluster
+	// metrics by external clients.
+	ClusterIDLabel = "cluster_id"
```

Comment:
`ClusterAnnotation` makes sense to me. `ClusterLabel` and `ClusterIDLabel` say nothing to me. I do not know what they are good for even with the godoc. What are "jobs managed by the prometheus-config-controller" and what are "guest cluster metrics by external clients"?

Reply:
`ClusterLabel` is the label we use internally to specify that a job should be managed by the `prometheus-config-controller`, as opposed to a job that is specified in the configmap.

ClusterIDLabel is the label we use for clients to access metrics for a guest cluster, such as desmotes.

I'll try to come up with some better names for them.

Reply:
How is the filtering of these labels be done? Do we filter in our code based on these labels or does Prometheus do it? Just wondering based on the labels vs annotations discussion we just had.

Reply:
These are Prometheus labels, not Kubernetes labels - they are attached to the Prometheus scrape configs. I don't think the labels vs. annotations discussion applies.

Reply:
Yeah you notice I have no idea how this thing works.

Reply:
Yeah, but I still love you, so it's okay <3